sci-astronomy/pal/files/pal-0.9.8-fix-strlcpy-musl.patch

```diff
@@ -0,0 +1,13 @@
+Bug: https://bugs.gentoo.org/898106
+bsd/string.h is available on most libc's
+--- a/palDfltin.c
++++ b/palDfltin.c
+@@ -119,7 +119,7 @@ static int ISBLANK( int c ) {
+ 
+ #endif
+ 
+-#ifdef HAVE_BSD_STRING_H
```

Comment:
I don't see the need for this patch, unless the `HAVE_BSD_STRING_H` test is failing on musl even when it's installed. But then we should fix the configure test.

Reply:
yes that seems to be the case Sam.